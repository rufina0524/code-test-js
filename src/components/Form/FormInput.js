import React from 'react';

const style = {
  display: "block"
};

const FormInput = (props) => {
  return (
    <form onSubmit={props.onSubmit}>
      <input style={style} type="text" name="title" placeholder="Client ID" />
      <input style={style} type="text" name="title" placeholder="Client Secret" />
      <input style={style} type="text" className="form-control" name="title" placeholder="Restaurant Name" />
      <input type="submit" value="Submit" />
    </form>
  )
};

export default FormInput;