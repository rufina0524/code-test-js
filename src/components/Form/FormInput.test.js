import React from 'react';
import FormInput from './FormInput';
import renderer from 'react-test-renderer';

describe('FormInput', () => {
  it('should match snapshot', () => {
    const comp = renderer
      .create(
        <FormInput
          type='text'
          style={{}}
          name={name}
          placeholder='test'
          onChange={jest.fn()}
        />
      )
      .toJSON();
    expect(comp).toMatchSnapshot();
  });
});