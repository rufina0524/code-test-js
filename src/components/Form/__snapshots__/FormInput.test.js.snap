// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormInput should match snapshot 1`] = `
<input
  name="nodejs"
  onChange={[MockFunction]}
  placeholder="test"
  style={Object {}}
  type="text"
/>
`;
